django/core/serializers/python.py

```diff
@@ -91,6 +92,7 @@ def Deserializer(object_list, **options):
     """
     db = options.pop('using', DEFAULT_DB_ALIAS)
     ignore = options.pop('ignorenonexistent', False)
+    handle_forward_references = options.pop('handle_forward_references', False)
```

Comment:
I still think that https://github.com/django/django/pull/6221#r68929405 is correct.  That is, automatically handling forward references changes behaviour and so `handle_forward_references` is required to maintain backwards compatibility.

The problem is that `serializers.deserialize` does not save data to the database, and so there's no point that `serializers.deserialize` can check to see whether there are incomplete objects -- this must be done by the caller of `serializers.deserialize`.